hack/update-version-list.sh

```diff
@@ -16,28 +16,42 @@
 
 set -euo pipefail
 
+VERSIONS_YAML_DIR=${VERSIONS_YAML_DIR:-"pkg/istioversions"}
 VERSIONS_YAML_FILE=${VERSIONS_YAML_FILE:-"versions.yaml"}
 HELM_VALUES_FILE=${HELM_VALUES_FILE:-"chart/values.yaml"}
 
 function updateVersionsInIstioTypeComment() {
-    selectValues=$(yq '.versions[].name | ", \"urn:alm:descriptor:com.tectonic.ui:select:" + . + "\""' "${VERSIONS_YAML_FILE}" | tr -d '\n')
-    versionsEnum=$(yq '.versions[].name' "${VERSIONS_YAML_FILE}" | tr '\n' ';' | sed 's/;$//g')
-    versions=$(yq '.versions[].name' "${VERSIONS_YAML_FILE}" | tr '\n' ',' | sed -e 's/,/, /g' -e 's/, $//g')
-    defaultVersion=$(yq '.versions[0].name' "${VERSIONS_YAML_FILE}")
+    selectValues=$(yq '.versions[].name | ", \"urn:alm:descriptor:com.tectonic.ui:select:" + . + "\""' "${VERSIONS_YAML_DIR}/${VERSIONS_YAML_FILE}" | tr -d '\n')
+    versionsEnum=$(yq '.versions[].name' "${VERSIONS_YAML_DIR}/${VERSIONS_YAML_FILE}" | tr '\n' ';' | sed 's/;$//g')
+    versions=$(yq '.versions[].name' "${VERSIONS_YAML_DIR}/${VERSIONS_YAML_FILE}" | tr '\n' ',' | sed -e 's/,/, /g' -e 's/, $//g')
+    defaultVersion=$(yq '.versions[1].name' "${VERSIONS_YAML_DIR}/${VERSIONS_YAML_FILE}")
```

Comment:
These commands were complex to begin with, so splitting the `VERSIONS_YAML` variable makes things much worse. Can we introduce `VERSIONS_YAML_PATH` to combine the two variables and use it here?